.coveragerc

```diff
@@ -1,4 +1,4 @@
 [run]
 branch = True
-include = */numpy/*
+source = numpy
```

Comment:
Should this be source = ., since the numpy package is in the repo root?

Reply:
I pushed a commit with your suggestion

Reply:
Looks like an identical result--probably need to play with it on a fork hooked up to codecov, although most previous debugging experiences with codecov were basically just sunk time.